src/background.js

```diff
@@ -2,17 +2,18 @@ const connections = {};
 
 chrome.runtime.onConnect.addListener(function (port) {
   var extensionListener = function (message, sender, sendResponse) {
```

Comment:
As I cant see, we are neither using this function out of scope, nor updating it further. So can you use const over var.

```suggestion
  const extensionListener = function (message, sender, sendResponse) {
```